README.md

```diff
@@ -1,2 +1,9 @@
-# eq-questionnaire-runner-deploy-image
-The Dockerfile used to build the EQ Questionnaire Runner base image.
+# EQ App Deploy Image
+The Dockerfile used to build a base image for deploying EQ containers with Helm.
+
+## Building
+The image can be built using:
+
+```sh
+docker build -t eq-app-deploy-image:latest
```

Comment:
I think this should use the Makefile command since we have it, or document both.